src/lib/Scenes/Home/Components/Sales/Components/LotsByFollowedArtists.tsx

```diff
@@ -1,120 +1,44 @@
 
 const DEFAULT_TITLE = "Lots by Artists You Follow"
-const PAGE_SIZE = 8 // FIXME: Increase
-
-interface RelayProps {
-  title?: string
-  relay: RelayPaginationProp
-  viewer: {
-    sale_artworks: {
-      pageInfo: object
-      edges: Array<{
-        node: object
-      }>
-    }
-  }
-}
 
 const Container = styled.View`padding: 10px;`
 
-const Query = graphql.experimental`
-  query LotsByFollowedArtistsQuery($count: Int!, $cursor: String) {
-    viewer {
-      ...LotsByFollowedArtists_viewer @arguments(count: $count, cursor: $cursor)
-    }
-  }
-`
-
-const Pagination = createPaginationContainer(
-  GridContainer,
-  graphql.experimental`
-    fragment LotsByFollowedArtists_viewer on Viewer
-      @argumentDefinitions(count: { type: "Int" }, cursor: { type: "String" }) {
-      sale_artworks(first: $count, after: $cursor, include_artworks_by_followed_artists: false)
-        @connection(key: "LotsByFollowedArtists_sale_artworks") {
-        pageInfo {
-          endCursor
-          hasNextPage
-        }
-        edges {
-          cursor
-          node {
-            is_biddable
-            artwork {
-              ...GenericGrid_artworks
-            }
-          }
-        }
-      }
-    }
-  `,
-  {
-    getConnectionFromProps: ({ viewer }) => viewer && viewer.sale_artworks,
-    getFragmentVariables: (prevVars, totalCount) => ({ ...prevVars, count: totalCount }),
-    getVariables: (props, { count, cursor }) => ({ count, cursor }),
-    query: Query,
-  }
-)
-
-export class LotsByFollowedArtists extends Component<any> {
-  render() {
-    return (
-      <QueryRenderer
-        environment={createEnvironment()}
-        query={Query}
-        variables={{
-          count: PAGE_SIZE,
-        }}
-        render={renderWithLoadProgress(Pagination, this.props)}
-      />
-    )
+interface Props {
+  isLoading?: () => boolean
+  title?: string
+  saleArtworks?: {
+    edges: Array<{
+      node: object | null
+    }> | null
   }
 }
 
-function GridContainer(props: RelayProps) {
-  const { relay, viewer, title = DEFAULT_TITLE } = props
+export function LotsByFollowedArtists(props: Props) {
+  const { isLoading, saleArtworks, title = DEFAULT_TITLE } = props
 
-  const artworks = get(viewer, "sale_artworks.edges", [])
+  const artworks = get(saleArtworks, "edges", [])
     .filter(({ node }) => !node.is_biddable)
     .map(({ node }) => node.artwork)
 
   if (!artworks.length) {
     return null
   }
 
-  const loadMore =
-    relay.hasMore() &&
-    once(() => {
-      relay.loadMore(PAGE_SIZE, x => x)
-    })
-
-  const handleScroll = ({ nativeEvent }) => {
-    if (isCloseToBottom(nativeEvent)) {
-      loadMore()
-    }
-  }
-
   return (
-    <ScrollView onScroll={handleScroll} scrollEventThrottle={30}>
+    <View>
       <SectionHeader title={title} />
       <Container>
         <GenericGrid artworks={artworks} />
+        {isLoading() && <Spinner style={{ marginTop: 20 }} />}
```

Comment:
In the interests of deadlines I went with this approach rather than trying to refactor `<InfiniteScrollArtworkGrid />` to handle content that is artist-generic and / or introduce complexity. Can return at a later point